deployment/nuget-keys.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Restoring NuGet Packages in CI
+page_title: Restoring NuGet Packages in CI
+description: "Get started with Telerik UI for Blazor and use NuGet Keys to authenticate with the Telerik NuGet server and restore packages in your CI or desktop environment."
+slug: installation/nuget-keys
+position: 7
+---
+
+# Restoring NuGet Packages in Your CI Workflow
+
+This article provides an overview of the most popular approaches for using token-based authentication to restore Telerik NuGet packages in your CI (Continuous Integration) workflow.
```

Comment:
```suggestion
This article provides an overview of the most popular ways to use token-based authentication to restore Telerik NuGet packages in your Continuous Integration (CI) workflow.
```

Reply:
For completeness, the article mentions one other way—basic authentication. But it handles in detail only the token-based authentication through NuGet keys and the workflows where these keys can be used.

```suggestion
This article provides an overview of the most popular ways for using token-based authentication to restore Telerik NuGet packages in your Continuous Integration (CI) workflow.
```